src/rdo.rs

```diff
@@ -78,14 +78,14 @@ impl RDOType {
 }
 
 #[derive(Clone)]
-pub struct RDOOutput {
+pub struct PartitionGroupParameters {
```

Comment:
To me, most guidance for naming for this struct will be that this struct can have
the results from multiple partitions, i.e. split partitions. And I believe, your new name "PartitioGroup" wanted to care that point.
As non native English user,  I hope adding "ing" to below struct name, i.e. "RDOPartitioningOutput" would fit for this struct! Since partitioning implies to me that in the middle of splitting partition, thus info from multiple partitions.

Reply:
To me that would not be an intuitive name at all, but I'd like to hear other opinions on this.